airflow/hooks/subprocess.py

```diff
@@ -35,24 +35,29 @@ def __init__(self) -> None:
         super().__init__()
 
     def run_command(
-        self, command: List[str], env: Optional[Dict[str, str]] = None, output_encoding: str = 'utf-8'
+        self,
+        command: List[str],
+        env: Optional[Dict[str, str]] = None,
+        output_encoding: str = 'utf-8',
+        bash_cwd: str = None,
```

Comment:
I feel this argument should simply be `cwd`. This matches the `subprocess` interface, which are probably familiar to many developers with Python experience.

Reply:
😄, that's right.
I change it now.

Reply:
Done